yarn.lock

```diff
@@ -3,1451 +3,1446 @@
 
 
 "@cable_ready/polyfills@^4.4.6":
-  version "4.4.6"
-  resolved "https://registry.yarnpkg.com/@cable_ready/polyfills/-/polyfills-4.4.6.tgz#57d242b3363902f4da3e76ed8c72d24d4a527f92"
-  integrity sha512-qdRzfUCx4Tal49XC1vHRxm0A7FxbBeF/7dOxmAFOqmXWVkO+pclk9v7KqK2G/OnYt7CyZrD/YxDV8rooog14Qg==
+  "integrity" "sha512-qdRzfUCx4Tal49XC1vHRxm0A7FxbBeF/7dOxmAFOqmXWVkO+pclk9v7KqK2G/OnYt7CyZrD/YxDV8rooog14Qg=="
+  "resolved" "https://registry.npmjs.org/@cable_ready/polyfills/-/polyfills-4.4.6.tgz"
+  "version" "4.4.6"
   dependencies:
```

Comment:
I'm pretty sure this file shouldn't have changed.

Reply:
These changes are the result of something running `npm install`. That probably means that @awisema needs to re-build the docker container, because it's using an old cached layer that uses `npm install`.